### PR TITLE
Feature/metrics template/creation of template for metrics framework

### PR DESCRIPTION
**Pull request by ksbn7**
Added updated metrics template to show usage of Micrometer Framework.

### CODE REVIEW

File: metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+2. `/metrics` generic endpoint
+
+[Reference: Metrics endpoint](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-metrics-endpoint)
+
+<details><summary>Response of /metrics</summary>
+<p>
+
+See example output below, showing many default metrics.
+Please also notice the custom metrics `devices.all.inventory.requests` and `device.inventory.requests`.
```
Comment (PestusAtSAG) [inline metrics/README.md:55]: I started the microservice locally and could only access http://localhost:8080/metrics/devices.all.inventory.requests. The device.inventory.requests i couldn't access...

**Review by PestusAtSAG — COMMENTED** [review]
Looks good, only accessing metrics endpoint /metrics/device.inventory.requests was not possible.  Is the endpoint not available in certain state or is this a bug?

File: metrics/README.md (hunk 1 above)
Comment (ksbn7) [reply metrics/README.md:55]: The "device.inventory.requests" is only accessible, once it actually got triggered.
Hence whenever you call the /devices/{deviceId} endpoint this metric is triggered and can be accessed.

In general metrics are only provided via /metrics endpoint once they got used at least once.

File: metrics/src/main/configuration/cumulocity.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	},
+	"isolation": "MULTI_TENANT",
+	"resources": {
+        "cpu": "1",
+        "memory": "512M"
+    },
+	"requiredRoles": [
+		"ROLE_INVENTORY_READ",
+		"ROLE_INVENTORY_CREATE",
+		"ROLE_INVENTORY_ADMIN"
```
[the remark this answers is not in the document]
Comment (ksbn7) [reply metrics/src/main/configuration/cumulocity.json:15]: Sure, missed that. Will fix it.

**Pull request merged**